themes/components-training-theme/src/css/variables/_colors.scss

```diff
@@ -0,0 +1,27 @@
+$primary: #F9A13B;
+$secondary: #96A2AE;
+
+$info: #2E5AAC;
+$success: #4AAB3B;
+$warning: #B95000;
+$danger: #DA1414;
+
+$info-lighten: #EEF2FA;
+$success-lighten: #EDF7EB;
+$warning-lighten: #FFF4EC;
+$danger-lighten: #FEEFEF;
+
+$primary:  #F9A13B;
+$secondary: #96A2AE;
+
+$white: #FFFFFF;
+
+$primary-hover: #FCA177;
+$primary-active: #F7890A;
+$primary-disabled: #8C8C8C;
+
+$secondary-hover: #FCA177;
+$secondary-active: #F7890A;
+$secondary-disabled: #8C8C8C;
+
+$primary-l3: clay-lighten($primary, 44.9) !default;
```

Comment:
`$primary-l3` , is this variable being used?

Reply:
I don't think so, I think I put it by mistake, I'll remove it

Reply:
Nice! Other than that, everything looks ok to me.